react/week3/todolist/src/components/timer.js

```diff
@@ -0,0 +1,17 @@
+import  React, { useState , useEffect } from 'react'
+export const DateTime = () => {
+    var [date,setDate] = useState(new Date());
```

Comment:
In general `var` is not used anymore, especially not with a React state. Make sure to use `const` for states. If you need to modify the value later of something other than a state, you can use `let`